src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    display: block;
+  }
+
+  &__title {
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 40px auto 0;
+  }
+
+  &__product {
+    line-height: 14px;
+    font-size: 10px;
+    color: $secondary-color;
+    margin: 4px 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  .star {
+    background-image: url(../images/star-active.svg);
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  .star--disabled {
+    background-image: url(../images/star.svg);
+  }
+
+  &__numbers {
+    color: $main-color;
+    font-size: 10px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+
+    .price {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      color: $main-color;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+
+    .price__title {
+      line-height: 18px;
+      font-size: 12px;
+      color: $secondary-color;
+    }
+  }
+
+  &__button {
+    color: #fff;
+    box-sizing: border-box;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    border: 1px solid #00acdc;
+
+    .button__title {
+      color: #00acdc;
+    }
```

Comment:
The selector `.button__title` is nested within `&__button:hover`. This suggests that `.button__title` is a child of `&__button`, but there is no such element in the HTML structure. Ensure that the HTML structure matches this SCSS or adjust the SCSS to match the actual HTML.